test/groovy/DockerExecuteOnKubernetesTest.groovy

```diff
@@ -91,15 +101,13 @@ class DockerExecuteOnKubernetesTest extends BasePiperTest {
     @Test
     void testRunOnPodNoContainerMapOnlyDockerImage() throws Exception {
         stepRule.step.dockerExecuteOnKubernetes(
-            script: nullScript,
-            juStabUtils: utils,
-            dockerImage: 'maven:3.5-jdk-8-alpine',
-            dockerOptions: '-it',
-            dockerVolumeBind: ['my_vol': '/my_vol'],
-            dockerEnvVars: ['http_proxy': 'http://proxy:8000'], dockerWorkspace: '/home/piper'
-        ){
-            bodyExecuted = true
-        }
+                script: nullScript,
```

Comment:
reformatting only. I would like to suggest not to have reformatting in such a PR in order not to blur the focus of the PR.